src/main/java/tc/oc/pgm/modules/SoundsMatchModule.java

```diff
@@ -32,51 +32,56 @@ public SoundsMatchModule createMatchModule(Match match) throws ModuleLoadExcepti
   }
 
   private static final Sound RAINDROP_SOUND = new Sound("random.levelup", 1f, 1.5f);
+  private static final Sound ENDERMAN_SCREAM = new Sound("mob.endermen.scream", 1f, 1.5f);
 
   public SoundsMatchModule(Match match) {
     super(match);
   }
 
-  private void playRaindrop(MatchPlayer player) {
+  private void playSound(MatchPlayer player, Sound sound) {
     if (player.getSettings().getValue(SettingKey.SOUNDS).equals(SettingValue.SOUNDS_ON)) {
-      player.playSound(RAINDROP_SOUND);
+      player.playSound(sound);
     }
   }
 
-  private void playRaindrop(MatchPlayerState playerState) {
-    playerState.getPlayer().ifPresent(this::playRaindrop);
+  private void playSound(MatchPlayerState playerState, Sound sound) {
+    playerState.getPlayer().ifPresent(player -> playSound(player, sound));
   }
 
-  private void playRaindrop(Competitor competitor) {
-    competitor.getPlayers().forEach(this::playRaindrop);
+  private void playSound(Competitor competitor, Sound sound) {
+    competitor.getPlayers().forEach(player -> playSound(player, sound));
   }
 
   @EventHandler(priority = EventPriority.MONITOR)
   public void onMatchPlayerDeath(MatchPlayerDeathEvent event) {
     ParticipantState killer = event.getKiller();
     MatchPlayer victim = event.getVictim();
-    if (killer != null
-        && !killer.getId().equals(victim.getId())
-        && !killer.getParty().getName().equals(victim.getParty().getName())) {
-      playRaindrop(killer);
+    if (killer != null) {
+      if (killer.getId().equals(victim.getId())) {
+
+      } else if (killer.getParty().getName().equals(victim.getParty().getName())) {
```

Comment:
Any reason not to use `killer.getParty().equals(victim.getParty())`?

Reply:
I can use that actually thanks for the heads up.